src/UsgsAstroLsPlugin.cpp

```diff
@@ -421,9 +423,22 @@ std::string UsgsAstroLsPlugin::convertISDToModelState(
    state.m_PlatformFlag = atoi(image_support_data.param("PLATFORM").c_str());
    state.m_AberrFlag = atoi(image_support_data.param("ABERR").c_str());
    state.m_AtmRefFlag = atoi(image_support_data.param("ATMREF").c_str());
-   state.m_IntTime = atof(image_support_data.param("INT_TIME").c_str());
    state.m_StartingEphemerisTime = atof(image_support_data.param("STARTING_EPHEMERIS_TIME").c_str());
    state.m_CenterEphemerisTime = atof(image_support_data.param("CENTER_EPHEMERIS_TIME").c_str());
+   if (image_support_data.param("NUMBER_OF_INT_TIMES").empty()) {

```

Comment:
This if could be changed to check if INT_TIME has more than one value, instead of looking for the NUMBER_OF_INT_TIMES parameter. It could also be changed to look for any value greater than 1.

Reply:
Since we are carrying a bunch of `NUMBER_OF_*` inside the ISD this is a 'canonical' implementation.  I would keep this way.